addons/mail/static/src/models/chat_window.js

```diff
@@ -166,18 +166,12 @@ registerModel({
          */
         onClickClose(ev) {
             ev.stopPropagation();
-            if (!this.exists()) {
-                return;
-            }
```

Comment:
this is a functional change, the `stopPropagation` will effectively no longer we called in this case

Reply:
This is a generic "problem" with this PR, all of the `preventDefault` and `stopPropagation` that were inside handlers that didn't have exists guard (or that were placed before them) will no longer be called.

I'm not sure if this is really an issue, but it's something to keep in mind.

Reply:
Yes, I'm aware of this change.
I think the previous code that made `stopPropagation` while not properly handling the event was wrong.
As for `preventDefault`, it might be an issue that click on `<a href="#">` does improper behaviour. I think it makes sense to use `t-on-click.prevent` for them instead. What do you think?

Reply:
I hate this syntax but we don't really have the choice.

I think we should do it also for `stopPropagation`, it doesn't really make sense to let it bubble in that case (if we decided that it should stop in the first place, which is another debate), if the record is dead the click should effectively be considered meaningless.